index.js

```diff
@@ -15,6 +15,11 @@ var PluginError = gutil.PluginError;
 var PLUGIN_NAME = 'gulp-istanbul';
 var COVERAGE_VARIABLE = '$$cov_' + new Date().getTime() + '$$';
 
+function getNormalizedPath(filepath) {
+  if (path.sep !== '\\') return filepath;
+  return filepath.replace(/\//g, '\\');
```

Comment:
`return filepath.replace(/\//g, path.sep);` then we don't have to worry about conditionals or anything.